packages/manager/apps/pci-private-registry/src/pages/create/useTranslatedBytes.tsx

```diff
@@ -0,0 +1,73 @@
+import { useTranslation } from 'react-i18next';
+
+// TODO: Add this filter in UX components
+const units = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+const unitsKibi = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB', 'ZiB', 'YiB'];
```

Comment:
```suggestion
const BYTE_UNITS = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'] as const;
const KIBI_UNITS = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB', 'ZiB', 'YiB'] as const;
```

I think it's better to see it's global constant and the `as const` specify to Typescript the value of array not change. (And not only the ref of array)

It's better for type check too